interaction-manager/src/lib_components/spatial-navigation.js

```diff
@@ -263,6 +279,30 @@ class SpatialNavigation {
         });
     }
 
+    /**
+     * Adds or override default direction keys with the specified ones
+     * @param {Object} customDirections - { up: 'W', left: 'A', right: 'D', down: 'S' }
+     * @param {Object} options - Optional settings.
+     * @param {Boolean} options.changeDefault - If true, overrides the default navigation keys. Defaults to false.
+     * @returns {void}
+     */
+    changeKeys(customDirections, options = { changeDefault: false }) {
+        if (Object.entries(customDirections).length === 0) return;
+
+        this.changeDefault = options.changeDefault;
+
+        const incorrectDirections = Object.keys(customDirections).filter(direction => !directions.includes(direction));
+        if (incorrectDirections.length > 0) return console.error(`The following directions: [${incorrectDirections.join(', ')}] you have entered are incorrect! `);
+
+        this.customKeys = {
+            ...this.customKeys,
+            ...customDirections,
+        };
+
+        this.removeKeyActions();

```

Comment:
Maybe we need to remove the previous custom keys here so we won't end at overwriting `up` position with `W` and `U` for example and then both keys are working for navigating up.

Reply:
Do as we discussed with Misho - the option `changeDefault` will clear all the keys so far - the default and the custom ones

Reply:
Fixed